lxatacstrategy.py

```diff
@@ -158,10 +160,18 @@ def transition(self, status, *, step):
             self.barebox.boot("")
             self.barebox.await_boot()
 
+            # The first boot takes quite some time because the eMMC is
+            # re-partitioned, filesystems are created and then the TAC reboots.
+            # Subsequent boots are faster and do not need the long timeout.
+            self.shell.login_timeout = 300 if self.first_boot else 60
+
```

Comment:
Just to make sure. Does this interfere with our internal task ``050.580 Watchdog sometimes resets the device during a reboot``? 

tbh: I don't think so. But I just want to be sure. :smiley:

Reply:
It could indeed hide the issue `050.580 Watchdog sometimes resets the device during a reboot` intends to solve and I have actually seen it happen when testing the change locally.

I am not sure what the expected boot time actually is.

Maybe we should close this PR, fix `050.580` and re-evaluate then.

Or we merge it and pinky swear that we will fix `050.580` anyways even if it does not bother us that much anymore.

Reply:
On the other side: For follow-up boots we reduce the boot timeout. So this may not mask the problem at all. I would suggest we merge this as is, fix `050.580` as planned.